diff --git a/build/src/utils/npm.js b/build/src/utils/npm.js
index a49faac0ba81f0a00d19e8427434cf013c9b4769..85956ad1f9b4819da0ff203904db3ce96011f4b0 100644
--- a/build/src/utils/npm.js
+++ b/build/src/utils/npm.js
@@ -60,7 +60,13 @@ function _stream() {
     return data;
 }
 function _tar() {
-    const data = /*#__PURE__*/ _interopRequireDefault(require("tar"));
+    const tarMod = require("tar");
+    const extract = tarMod.extract ?? (tarMod.default && tarMod.default.extract);
+    const data = {
+        default: {
+            extract
+        }
+    };
     _tar = function() {
         return data;
     };
